DEPLOYMENT.md

```diff
@@ -10,13 +10,14 @@ If setting up a new environment `ENV_NAME` for the first time:
 - Add a new AWS configuration locally
 - Add it to `S3_BUCKET_SUFFIXES` in `.env`
 
-## 2. Deploy the contracts:
+## 2. Deploy and verify the contracts:
```

Comment:
i would also update the .env.example to

```
INFURA_KEY=
ETHERSCAN_API_KEY=
ALCHEMY_KEY=
S3_BUCKET_PREFIX=argent-smartcontracts
S3_BUCKET_SUFFIXES=dev:hydrogen
S3_CONFIG_KEY=backend/config.json
DEPLOYER_GAS_PRICE=
```

plus the correct values for S3_BUCKET_SUFFIXES

it took me hours to find out this values